server/dar-server/package.json

```diff
@@ -12,6 +12,6 @@
     "express": "^4.16.2"
   },
   "peerDependencies": {
-    "pubsweet-server": "^8.0.0"
+    "pubsweet-server": "^7.0.0"
```

Comment:
Don't think we should be downgrading these - probably what is causing the failures in the pipeline.